### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by samhstn**
Addresses #1 

Initial setup:
- [x] Travis
- [x] Code coverage
- [x] Linter
- [x] Beautify Readme (Badges :1st_place_medal:, etc)
- [x] Build first iteration of amp converter
- [x] Publish on [hex](https://hex.pm/)

### CODE REVIEW

**Comment by samhstn** [comment]
Initial documentation published at:
https://hexdocs.pm/amp/0.1.0
Available on hexdocs:
https://hex.pm/packages/amp

File: README.md
Comment (nelsonic) [inline README.md:11]: http://rhodesmill.org/brandon/2012/one-sentence-per-line

File: README.md
Comment (nelsonic) [inline README.md:17]: long lines are harder to update/review.
keep it to one sentence per line for readability.
http://rhodesmill.org/brandon/2012/one-sentence-per-line

**Review by nelsonic — APPROVED** [review]
@shouston3 this looks _really_ good to me. 
passing to @jruts for second review. 👍

File: README.md
Comment (samhstn) [reply README.md:17]: Thanks for the review, I'll know for next time 👍

**Review by jruts — APPROVED** [review]
Nothing to comment. It looks very good to me :+1:

**Comment by jruts** [comment]
@nelsonic you will have to merge, no rights to do so :)

**Comment by iteles** [comment]
@jruts Thanks for the review! I'll merge this one but just so you know: core team now has admin access to this repo :tada:

**Pull request merged**
